test/parallel/test-events-once.js

```diff
@@ -17,7 +17,7 @@ async function onceAnEvent() {
     ee.emit('myevent', 42);
   });
 
-  const [value] = await once(ee, 'myevent');
+  const [value] = await once(ee, 'myevent', null);
```

Comment:
Any reason not to create a new test case rather than altering an existing one? Especially considering that this code appears in the documentation as well?

Reply:
I also wonder if this isn't a code smell in the lib code. I'd be inclined to remove the default parameter and then passing explicit `null` to test both branches of the ternary becomes unnecessary.

Reply:
(The other added tests look good to me.)

Reply:
In order to improve the code coverage when options is null(https://coverage.nodejs.org/coverage-0b6d3070a176d437/lib/events.js.html#L712). Now I realize passing `null` or using the default parameters is different behaviors.

Thanks for the guidance, New test cases have been added.